install/templates/step1.php

```diff
@@ -0,0 +1,35 @@
+<?php include __DIR__ . '/header.php'; ?>
+
+    <h2><?= _('Welcome to the installation of Poweradmin!') ?></h2>
```

Comment:
I'm just wondering if short php tags are enabled by default in various Linux distributions?